packages/sdk/src/AbstractConfigManager.ts

```diff
@@ -302,7 +302,7 @@ export abstract class AbstractConfigManager {
             const newConfig: IConfig = await ConfigBuilder.build(impConfig, global, opts);
             config.api.layers.merge(newConfig);
             await config.save(false);
-        } catch (err) {}
+        } catch (_err) {}
```

Comment:
The error variable can be omitted here since it's unused:
```suggestion
        } catch {}
```